### DISCUSSION TITLE
Category Image - Where or How can this be displayed?

### DISCUSSION THREAD

**Discussion by gnossosp** [openstore-ecommerce/OpenStore#177] [Q&A]
There is a category image upload in Catalog>Categories>Image. I'm unsure of the purpose or where I can display this image. 

There is a "Category Message" editor and I have un commented the @(Category(navigationdata, "message", Model)) item in ProductDisplayAjaxList.chstml to display the message, however I don't see if there is a way to display the Category Image in a similar manner.

Can anyone enlighten me on how/where to display Category Image on product display list area?
Thanks

**Comment by DNNMonster** [comment]
You can use the image wherever you like when you display your categories.  It's optional.

For the product list display you might take a look at the ProductDisplayListBody.cshtml file and and use the GetDefaultCategory() method of the ProductData class.  I think it may have the image url you're interested in.

`product.GetDefaultCategory().imageurl`
